scripts/cnv_wdl/cnv_common_tasks.wdl

```diff
@@ -224,6 +273,7 @@ task CollectCounts {
     output {
         String entity_id = base_filename
         File counts = counts_filename
+        File counts_idx = if enable_indexing_ then "${base_filename}.${counts_index_filename_extension}" else "/dev/null"
```

Comment:
Not sure if there is a better way to handle optional outputs...can reviewer check?

Reply:
Oops, forgot to do this one.

Reply:
Hmm, let me just remove this output for now.  Downstream WGS gCNV WDLs can force indexing for the time being.  We can try to move towards tsv.gz + indexing being the only behavior allowed by this task, if not the Java code paths.